tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -1,5 +1,6 @@
 """
-Test to measure pvc scale creation & deletion time. Total pvc count would be 1500
+Test to measure pvc scale creation & deletion time. Total pvc count would be
+500 times the number of pvcs
```

Comment:
do you mean `500 times the number of Worker Nodes` ?

Reply:
yes. i meant worker nodes